### ISSUE TITLE
Feature suggestion: Email Reports

### ISSUE THREAD

**Issue by Tragio** [ibericode/koko-analytics#17]
Send a weekly or monthly e-mail report to the client with all the most important information. This is important for those clients that don't access the dashboard frequently or ever. Thank you for your amazing work!!

---

_Please vote on this feature suggestion using :+1: or :-1: below._

**Comment by dannyvankooten** [comment]
Hey @Kulcanhez,

Great suggestion, thank you for your contribution!

**Comment by umutusu** [comment]
@dannyvankooten Any news on this?

**Comment by craigcpaterson** [comment]
This would be a really helpful addition. Please consider adding it @dannyvankooten

**Comment by 12nick12** [comment]
This would be awesome :-)

**Comment by lokacoding** [comment]
Any updates on this one? That would be a killer feature.

**Comment by dannyvankooten** [comment]
Update: we're retiring our [OpenCollective initiative](https://opencollective.com/koko-analytics) in favor of a Pro version of the plugin containing some of the more advanced features like custom event tracking and email reports. 

- Custom event tracking is currently in a working state and can be viewed in the [live demo](https://www.kokoanalytics.com/?go-to-demo) here. 
- Periodic email reports with a customizable email template is up next! 

So in short, expect a ton of progress on this over the next few weeks. Finally.

**Comment by dannyvankooten** [comment]
This feature suggestion is now tracked and further discussed in #199
